indexing-hadoop/src/main/java/io/druid/indexer/HadoopDruidIndexerMapper.java

```diff
@@ -77,6 +80,7 @@ protected void map(
       }
       catch (Exception e) {
         if (config.isIgnoreInvalidRows()) {
+          log.warn(e, "Ignoring invalid row [%s] due to parsing error", value.toString());
```

Comment:
I have mixed feelings about that limit.

In our architecture, the realtime is treated as "Probably right" and the batch fixup is treated as "definitely right" 

As such, If the data itself is bad, It should probably be cleaned up externally unless the user is really OK with completely loosing data in Druid.

Thinking a bit more about it, I can see two major ways of looking at it:

1. `ignoreInvalidRows` is an undocumented option for people who are intentionally trying to loose data. In which case either printing out no logs or printing out only a few lines would be OK.
2. `ignoreInvalidRows` is a "feature" whereby bad rows can go to an ephemeral something^(tm) where they can be fixed and recovered.

But officially supporting `ignoreInvalidRows` feels like it breaks the Lambda assumption that batch fixup gets all the data. As such, I'm suggesting #1, which is how this PR is going towards anyways.